### DISCUSSION TITLE
Portugol Studio não instala no macOS Catalina 10.15.7

### DISCUSSION THREAD

**Discussion by livvlima** [UNIVALI-LITE/Portugol-Studio#964] [Problemas]
Quando tento instalar o programa em meu computador, recebo esta mensagem:
![Captura de Tela 2021-07-28 às 13 56 54](https://user-images.githubusercontent.com/87501571/127364685-9b5d2176-213b-49b2-92b6-fe796c197ee0.png)

Gostaria de saber como solucionar este problema, tendo em vista que preciso deste programa funcionando para acompanhar algumas aulas da faculdade.

Obrigada.

**Comment by AdsonEsteves** [comment]
@livvlima 
Tenta clicar com o botao direito na caixa e ir em "Abrir"
Tente isso mais de umas duas ou três vezes.

**Comment by travincas** [comment]
Não consigo instalar no sistema Mac os Monterey. Informa que a imagem está danificada, sendo que já baixei até versões posteriores. Não está bloqueando. Com o Catalina funciona normal. Como resolverei isso se recorrer a realizar o downgrade?

**Reply by GeorgeAGomes** [reply]
Consegui instalar mas ao abrir ainda ocorre um erro e o app fecha automaticamente

Abra o terminal
digite "cd Downloads" ou o caminho onde o arquivo esta salvo
hdiutil attach portugol-studio-2.7.5-osx.dmg
instale

caso apareça um aviso "cannot be opened because it is from an unidentified developer"

Abra preferencias do sistema -> Segurança e Privacidade -> Clique no cadeado e coloque a senha
Na aba general clicar para liberar a instalação

Proseguir instalação

**Comment by AdsonEsteves** [comment]
@travincas 
Ainda não descobrimos como resolver esse problema infelizmente.
Se quiser pode tentar o https://portugol-webstudio.cubos.io/, porém é mais limitado.
